erpnext/hr/doctype/leave_encashment/leave_encashment.py

```diff
@@ -64,6 +64,9 @@ def get_leave_details_for_encashment(self):
 
 		allocation = self.get_leave_allocation()
 
+		if not allocation:
+			frappe.throw("No Leaves Allocated to Employee: {0} for Leave Type: {1}".format(self.employee, self.leave_type))
```

Comment:
@Anurag810 Where is multi-lingual support for the message? This is bad :(

Reply:
Done